spec/values/calculation/hypot.hrx

```diff
@@ -0,0 +1,241 @@
+<===> README.md
+Most of the same behavior tested for `calc()` applies to `hypot()`, but for
+terseness' sake isn't tested explicitly.
+
+<===>
+================================================================================
+<===> units/none/input.scss
+a {b: hypot(3, 4, 5, 6, 7)}
+
+<===> units/none/output.css
+a {
+  b: 11.6189500386;
+}
+
+<===>
+================================================================================
+<===> units/compatible/input.scss
+a {b: hypot(13cm, 4mm, 5q, 6in, 7px)}
+
+<===> units/compatible/output.css
+a {
+  b: 20.0366545892cm;
+}
+
+<===>
+================================================================================
+<===> units/real_and_unknown/input.scss
+a {b: hypot(13cm, 4%)}
+
+<===> units/real_and_unknown/output.css
+a {
+  b: hypot(13cm, 4%);
+}
+
+<===>
+================================================================================
+<===> units/unknown/input.scss
+a {
+  b: hypot(1%, 2%);
+}
+
+<===> units/unknown/output.css
+a {
+  b: hypot(1%, 2%);
+}
+
+<===>
+================================================================================
+<===> units/fake/input.scss
+a {
+  b: hypot(1foo, 2bar);
+}
+
+<===> units/fake/output.css
+a {
+  b: hypot(1foo, 2bar);
+}
+
+<===>
+================================================================================
+<===> units/real_and_fake/input.scss
+a {
+  b: hypot(1px, 2bar);
+}
+
+<===> units/real_and_fake/output.css
+a {
+  b: hypot(1px, 2bar);
+}
+
+<===>
+================================================================================
+<===> simplification/input.scss
+@use "sass:math";
+a {b: hypot(3 + 2, 4 * 10, 5 * 40, math.div(6, 2.54), 7 * math.div(96, 2.54))}
+
+<===> simplification/output.css
+a {
+  b: 334.1051331579;
+}
+
+<===>
+================================================================================
+<===> infinity/first/input.scss
+a {b: hypot(infinity, 1, 1)}
+
+<===> infinity/first/output.css
+a {
+  b: calc(infinity);
+}
+
+<===>
+================================================================================
+<===> infinity/second/input.scss
+a {b: hypot(1, infinity, 1)}
+
+<===> infinity/second/output.css
+a {
+  b: calc(infinity);
+}
+
+<===>
+================================================================================
+<===> percentage/input.scss
+a {b: hypot(10%, 5%)}
+
+<===> percentage/output.css
+a {
+  b: hypot(10%, 5%);
+}
```

Comment:
This should be with the `units` tests. Generally, different test cases should have as consistent a structure between them as possible (given that they sometimes have different testing needs). Same goes for other unit tests here.